benchadapt/python/benchadapt/result.py

```diff
@@ -35,13 +35,23 @@ class BenchmarkResult:
         Measurement data and summary statistics. If ``data`` (a list of metric values),
         ``unit`` (for that metric, e.g. ``"s"``), and ``iterations`` (replications for
         microbenchmarks) are specified, summary statistics will be filled in server-side.
+    error : Dict[str, Any]
+        A dict containing information about errors raised when running the benchmark. Any
+        schema is acceptable, but may contain stderr, a traceback, etc.
+    validation : Dict [str, Any]
+        Benchmark results validation metadata (e.g., errors, validation types).
     tags : Dict[str, Any]
         Many things. Must include a ``name`` element (i.e. the benchmark name); often
         includes parameters either as separate keys or as a string in a ``params`` key.
         If suite subdivisions exist, use a ``suite`` tag. Determines history runs.
     info : Dict[str, Any]
         Things like ``arrow_version``, ``arrow_compiler_id``, ``arrow_compiler_version``,
         ``benchmark_language_version, ``arrow_version_r``
+    optional_benchmark_info : Dict[str, Any]
+        Optional information about Benchmark results (e.g., telemetry links, logs links).
+        These are unique to each benchmark that is run, but are information that aren't
+        reasonably expected to impact benchmark performance. Helpful for adding debugging
+        or additional links and context for a benchmark (free-form JSON)
```

Comment:
I love all of the clarity here

Reply:
ha, I just copied these directly from the swagger